### DISCUSSION TITLE
Trackball emulation

### DISCUSSION THREAD

**Discussion by AeliusSaionji** [averycblack/mac-precision-touchpad#3] [Ideas]
Hello!

I've been trying to get this back in some form ever since Synaptics removed it from drivers circa 2009. They used to call it Momentum.

Steam uses this concept on their Steam Controller and Steam Deck touchpad.

This feature allows the user to "push" the cursor as if you were operating a trackball mouse. You get the same benefit as trackballs: less wrist movement, less wrist strain. It also gives you more precision, because you can keep the cursor sensitivity low for normal tracking, while still being able to effortlessly jump large distances by flicking the cursor.

This video starts with a visual demonstration. https://www.youtube.com/watch?v=jWmzlJkMiMw

An important point to note is that, after a flick, you can stop the cursor by putting your finger on it. Just as you can stop a real trackball ball from rolling.

If this is out of the project scope, I understand.

Thanks!

**Comment by averycblack** [comment]
I think this is out of scope for this project. It does sound pretty cool though, bet you could do something elsewhere in Windows.
